app/views/pwa/service_worker.js

```diff
@@ -9,10 +9,19 @@ const pushHandler = async (event) => {
     {
       body: notificationData.body,
       icon: notificationData.icon,
+      data: notificationData.data,
     }
   );
   event.waitUntil(notificationPromise);
 };
 
+const notificationclickHandler = async (event) => {
```

Comment:
```suggestion
const notificationClickHandler = async (event) => {
```

Reply:
Yes, I know what you mean.
But original event name is `notificationclick`...

https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope/notificationclick_event